.github/workflows/build-electron.yml

```diff
@@ -56,3 +56,43 @@ jobs:
       with:
         name: invest-workbench-${{ matrix.os }}
         path: dist/invest-workbench_*
+
+    - name: Set variables for GCS deploy target
+      shell: bash
+      # TODO: If this is a release, don't include the commit hash
+      run: |
+        echo "PKG_VERSION=$(jq -r .version package.json)" >> $GITHUB_ENV
+        echo "GIT_REV=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+        echo "BUCKET=$([ github.actor == 'natcap' ] \
+          && echo 'gs://releases.naturalcapitalproject.org/invest-workbench/' \
+          || echo 'gs://natcap-dev-build-artifacts/invest-workbench/${{ github.actor }}' \
+          )" >> $GITHUB_ENV
```

Comment:
Is this the same as `export PKG_VERSION=$(jq -r .version package.json)`?

Reply:
Found it here: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
@dcdenu4 more/less the same I guess. I think I was following some GHA docs...let me see...https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable